share/dkg/dkg.go

```diff
@@ -460,7 +462,7 @@ func (d *DistKeyGenerator) ProcessDeals(bundles []*DealBundle) (*ResponseBundle,
 		// if the node is evicted, we don't even need to send a complaint or a
 		// response response since every honest node evicts him as well.
 		// XXX Is that always true ? Should we send a complaint still ?
```

Comment:
comment still valid?

Reply:
The XXX is probably not relevant but I'll take the time to dive more into it - I'm clearing issues from the report "one by one".